go.mod

```diff
@@ -37,6 +37,7 @@ require (
 	go.opencensus.io v0.22.4 // indirect
 	golang.org/x/crypto v0.0.0-20210220033148-5ea612d1eb83 // indirect
 	golang.org/x/net v0.0.0-20210316092652-d523dce5a7f4
+	gopkg.in/yaml.v2 v2.4.0 // indirect
```

Comment:
Why we need an older version of the `yaml` module?

Reply:
Sorry, my bad. I was using v2 in the file 'additional.go' 🤣. I already removed it.